packages/flutter/lib/src/widgets/image.dart

```diff
@@ -44,6 +45,15 @@ ImageConfiguration createLocalImageConfiguration(BuildContext context, { Size si
   );
 }
 
+/// Prefetches an image into the image cache.
+Future<Null> prefetchImage(ImageProvider provider, BuildContext context, { Size size }) {
+  final ImageConfiguration config = createLocalImageConfiguration(context, size: size);
```

Comment:
Having to pass `size` seems a bit awkward. Maybe this can be a method on the `Image` class?